### ISSUE TITLE
tflite micro softmax op is still version 1

### ISSUE THREAD

**Issue by fklabach** [tensorflow/tensorflow#35748]
@tensorflow/micro

**System information**
- Host OS Platform and Distribution: Ubuntu 18.04
- TensorFlow installed from: Source
- Tensorflow version: 4b3c1199a97cb36b8866d98e7036f4ec3e70abd6
- Target platform: Apollo3

**Describe the problem**

The tflite micro softmax op in [tensorflow/lite/micro/kernels/softmax.cc](https://github.com/tensorflow/tensorflow/blob/4b3c1199a97cb36b8866d98e7036f4ec3e70abd6/tensorflow/lite/micro/kernels/softmax.cc) already has int8 input support.
From what I understand this should be version 2 in [tensorflow/lite/micro/kernels/all_ops_resolver.cc](https://github.com/tensorflow/tensorflow/blob/4b3c1199a97cb36b8866d98e7036f4ec3e70abd6/tensorflow/lite/micro/kernels/all_ops_resolver.cc#L26)

**Comment by njeffrie** [comment]
Thanks for reporting this bug.  I have landed a patch to fix this operator as well as a handful of others internally and it should show up on Github soon.
